core/core/models/payload.py

```diff
@@ -25,7 +25,10 @@ class PhonePayload(BaseModel):
     phone: str
     old_phone: Optional[str] = None
 
-
+    def __init__(self):
+   #     validate_phone_number(self.phone)
+        pass
```

Comment:
I will double-check, but I guess that pydantic (the library that we used here) has its own way to validate models.

Reply:
Okay. I'll remove this part of the code. If it needed we return to it during the US-19 implementation